src/abyssal_tome/model.py

```
@@ -130,47 +140,61 @@ def load_rulings_data(file_path_str: str = "../assets/processed_rulings_v3_ai_en
                 ALL_RULINGS_DATA[ruling_obj.id] = ruling_obj
                 loaded_count += 1
             except Exception as e:
-                logging.error(f"Error validating ruling data for ID {ruling_dict.get('id', 'N/A')}: {e}\nData: {ruling_dict}") # Added N/A default
+                logging.error(
+                    f"Error validating ruling data for ID {ruling_dict.get('id', 'N/A')}: {e}\nData: {ruling_dict}"
+                )  # Added N/A default
```

Ruff (0.11.9)

`error()` call on root logger

(LOG015)